libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -2091,6 +2091,18 @@ uint16_t AP_Mission::get_index_of_jump_tag(const uint16_t tag) const
     return 0;
 }
 
+#if AP_SCRIPTING_ENABLED
+bool AP_Mission::get_last_jump_tag(uint16_t &tag, uint16_t &age) const
```

Comment:
needs a comment for units of age (the name makes it sound like a time, which is isn't)